src/app/modules/consultations/consultation-profile/consultation-questionnaire/consultation-questionnaire.component.ts

```diff
@@ -181,6 +182,43 @@ export class ConsultationQuestionnaireComponent
     });
   }
 
+  getQuestionText(question) {
+    this.currentLanguage = this.cookieService.get('civisLang');
+
+    if (this.currentLanguage === 'hi') {
+      const questionTextHindi = question?.hindiQuestionText;
+      if (questionTextHindi) {
+        return questionTextHindi;
+      }
+    }
+
+    if (this.currentLanguage === 'or') {
+      const questionTextOdia = question?.odiaQuestionText;
+      if (questionTextOdia) {
+        return questionTextOdia;
+      }
+    }
+
+    return question?.questionText;
+  }
```

Comment:
Similar logic is being repeated multiple times. Move the logic to a shared function and pass variable data as arguments